src/js/common/schemaform/definitions/pciuAddress.js

```diff
@@ -0,0 +1,38 @@
+import PCIUAddressField from '../fields/PCIUAddressField';
```

Comment:
Good point @jbalboni, as far as the forms are concerned this is the only instance that uses PCIU. Letters is another example, and it looks like profile may be another in the future, but neither are via schemaform. I will move.

Reply:
Also, I think it may be a goal to create a common PCIU address component to be potentially used by letters and profile, is there already a ticket for this @markgreenburg @cvalarida @ncksllvn? If not, should one be set up?

Reply:
There is a form component for creating/editing Addresses used by both Letters and Profile at `Letters/components/Address.jsx`. I think it could use some updating but it's still a good place to consolidate address forms. We also have some address formatters for the Profile work for displaying street/city/state/zip consistently across the different address formats that we could easily separate out into a utility file. Would that help at all?

Reply:
Right now it's in a React component but not being exported. It's located [here](https://github.com/department-of-veterans-affairs/vets-website/blob/7b63ad24e570f1733fb2dda690d1ff68c9d120a2/src/js/personalization/va-profile-beta/components/AddressSection.jsx#L65).

Reply:
Thanks for the context @ncksllvn, it would be great to reuse the address formatter as a utility when displaying the addresses in the 526 form, there is at least one difference with the 526 forwarding address (it includes an effective date), but that could probably be displayed beneath the address view.

Reply:
@cehsu absolutely, will do! I'll get a PR open shortly and add you as a CR if that's okay.

Reply:
Thanks @ncksllvn, sounds good!

Reply:
Actually, @ncksllvn, perhaps we should wait until @cvalarida and @markgreenburg weigh in, as we currently present more than just the address information (email, phone, etc.), and currently the forwarding address UI is presented beneath all of the other information. I'm not sure whether the display mode needs to be consistent with the input mode, if not, reusing what you've already built for the profile would be great.

Reply:
Also there is an [open question](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/9278) for @goldenmeanie regarding how the input mode should be set up, so perhaps my earlier point is a non-issue, if it is alright to move the forwarding address up right beneath the mailing address.